index.js

```diff
@@ -1,5 +1,5 @@
 import { writeFile } from 'node:fs/promises'
-import { dirname, resolve } from 'path'
+import { dirname, resolve, sep } from 'node:path'
```

Comment:
you can update ESLint config and it will check `node:` prefix automatically (if you want, we can do it later)

Reply:
Planned
https://github.com/evilmartians/astro-typedoc/issues/27